R/RodentData.R

```diff
@@ -1,117 +1,95 @@
-#' Length of non-missing values in a vector
+#' @importFrom stats na.omit
+
+#' @title Length of non-missing values in a vector
+#'
+#' How many values in the given vector are not NAs
 #'
 #' @param x vector of values
 #' @return integer of how many non-NA values in x
 #'
 #' @export
 #'
-true_length <- function(x){
- length(which(!is.na(x)))
-}
-
-#' Join plots and trapping tables
-#'
-#' @param trapping trapping_table from Portal data (can be filtered)
-#' @param plots plots_table from Portal data
-#' 
-#' @return trapping table with sampled column removed and treatment column 
-#'   added 
-#'
-#' @export 
-#'
-join_plots_to_trapping <- function(trapping, plots){
```

Comment:
`join_plots_to_trapping` is removed here, but is still used in the codebase (e.g. line 223 within this script)... did it get moved to a different script or does it need to be reinstated here?

Reply:
Ah, I moved it into `data_processing.R` to be with the other `join_XX_to_YY` function definitions.

Reply:
oh! dang github had shrunk `data_processing.R` so i couldn't see the code in it, i'll check it out.